website/docs/terms/deploying.md

```diff
@@ -0,0 +1,20 @@
+---
+id: deploying
+title: Deploying
+description: Deploying dbt in production means setting up a system to run a dbt job on a schedule, rather than running dbt commands manually from the command line.
+displayText: Deploying
+hoverSnippet: Deploying dbt in production means setting up a system to run a dbt job on a schedule, rather than running dbt commands manually from the command line.
+---
+
+Deploying dbt in production means setting up a system to run a dbt job on a schedule, rather than running dbt commands manually from the command line.
```

Comment:
```suggestion
Deploying dbt in production means setting up a system to run a dbt job on a schedule, rather than running dbt commands manually from the command line. For more details, refer to [Deploy dbt jobs](/docs/deploy/deployments). 
```